### DISCUSSION TITLE
There is open mt7668-wifi-bt driver available. Is it possible to integrate it for A95X F2 (Mediatek)?

### DISCUSSION THREAD

**Discussion by votarum** [devmfc/debian-on-amlogic#44] [General]
Hi! There is open mt7668-wifi-bt driver available - https://github.com/noob404yt/mt7668-wifi-bt
Is it possible to integrate it for A95X F2 (Mediatek)?

**Comment by devmfc** [comment]
Yes, sure, ik you get it to work, tell me how you did it and I will put it in the next release. 

The kernel headers are included in the release files, you will probably need those. 

Thanks!
